gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannelTest.java

```diff
@@ -360,16 +359,6 @@ public void writeHandlesErrorOnInsertRequestWithoutUncommittedData() throws Exce
     assertThrows(IOException.class, writeChannel::close);
   }
 
-  @Test
```

Comment:
Looks like we are removing tests, instead of adding to them. Please add the reasoning, preferable more test coverage

Reply:
Modified the earlier removed test case. Here the first upstream write succeeds, but resulting downstream write to GCS fails. Hence when the second write is called, it should raise an exception

Reply:
Add similar test for close call too.

Reply:
There are several for the `close` such as `writeHandlesErrorOnInsertRequestWithUncommittedData`